### PR TITLE
Updated README.md with Amine Ersu profile information

### PR DESCRIPTION
**Pull request by ersuamine**

### CODE REVIEW

**Comment by zeynepcakm** [comment]
such a good job Amine! thank you for your time and effort!

**Review by zeynepcakm — COMMENTED** [review]
you've done so much, thank you!

**Comment by 4745Mthn** [comment]
**_Thank you_**

File: README.md [hunk 1]
```diff
@@ -32,9 +32,11 @@
 
 ## Amine Ersu
 
-![Profile Picture](profile-picture-url)
```
Comment (4745Mthn) [inline README.md:35]: Thanks

**Review by 4745Mthn — COMMENTED** [review]
thanks

**Review by 4745Mthn — APPROVED** [review]

**Pull request merged**